source/releasenotes/2024-06-12

```diff
@@ -0,0 +1,7 @@
+---
+title: "Auth0 Universal Login"
+published_date: "2024-06-12"
+categories: [general, security]
+---
+
+Pantheon is enhancing the new user account creation and management process to offer a more stable, modern, and secure first experience. We are moving the registration and login experience to utilize Auth0's Universal Login feature.
```

Comment:
@kelly-mcgonnigle can you add more detail here. Maybe before/after screenshots or links. I don't know, and I don't think most customers will know what "Auth0's Universal Login feature" means